test/spacebomb.test.did

```diff
@@ -18,20 +18,6 @@ assert blob "DIDL\04\6c\03\01\7f\02\01\03\02\6c\01\01\70\6c\00\6d\00\01\03\80\94
 assert blob "DIDL\02\6d\01\6d\7f\01\00\05\ff\ff\3f\ff\ff\3f\ff\ff\3f\ff\ff\3f\ff\ff\3f"    !: () "vec vec null (extra argument)";
 assert blob "DIDL\03\6c\01\d6\fc\a7\02\01\6d\02\6c\00\01\00\80\ad\e2\04"                   !: () "vec record {} (extra argument)";
 
-// Messages with exactly 2_000_000 zero-length elements should succeed
```

Comment:
By dropping those tests, the above comment "Messages with more than 2_000_000 zero-length elements in vectors should be rejected" makes no sense anymore.